templates/bot/ts/default/commands/SSOCommand.ts

```diff
@@ -0,0 +1,50 @@
+import { IBotCommand } from "./IBotCommand";
+import { ResponseType } from "@microsoft/microsoft-graph-client";
+import { CardFactory } from "botbuilder";
+import { createMicrosoftGraphClient, OnBehalfOfUserCredential } from "@microsoft/teamsfx";
+import { DialogContext } from "botbuilder-dialogs";
+
+export class SSOCommand implements IBotCommand {
+  public commandKey = "show";
+
+  async run(parameters: any): Promise<any> {
+    const ssoDialog = parameters.ssoDialog;
+    ssoDialog.commandHandler = this.showUserInfo;
+    await ssoDialog.run(parameters.context, parameters.dialogState);
+  }
+
+  async showUserInfo(stepContext: DialogContext, ssoToken: string) {
```

Comment:
What's the return type of this method?

Reply:
no return of this method

Reply:
One of the best practice of TS is to declare return type of method. If there is nothing to return, you can just use Promise<void> as return type